### PR TITLE
add task 11-refactor-get-api-articles-article_id

### PR DESCRIPTION
**Pull request by jorgemf2604**
This is a PR for task 11-refactor-get-api-articles-article_id

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -159,6 +159,7 @@ describe("App", () => {
           expect(article.title).toBe("Eight pug gifs that remind me of mitch");
           expect(article.article_id).toBe(3);
           expect(article.author).toBe("icellusedkars");
```
Comment (sirKevlar) [inline __tests__/app.test.js:182]: We shouldn't amend existing tests. The existing tests should still pass if they are robust enough (which they should be) Instead this should be a whole new test whose only job will be to check if there is now a comment_count key/value pair

Comment (jorgemf2604) [reply __tests__/app.test.js:182]: done

File: models/articles-model.js [hunk 2]
```diff
@@ -44,9 +44,28 @@ const fetchAllArticles = async (
...
+//         return Promise.reject({
+//           status: 404,
+//           msg: "Article not found",
+//         });
+//       } else {
+//         return articles[0];
+//       }
+//     });
+// };
+
```
Comment (Slyose) [inline models/articles-model.js]: You shouldn't leave comments like this in when it comes to actually submitting the PR
If you want to keep notes of old chunks of code, consider making a file and adding it to .gitignore so it doesnt get pushed up to hithub

**Review by Slyose — APPROVED** [review]
Just needs a commented out chunk of code removed, then feel free to merge without another PR! ✅

**Pull request merged**